build-info-api/src/main/java/org/jfrog/build/api/IssueTracker.java

```diff
@@ -37,4 +38,22 @@ public String getVersion() {
     public void setVersion(String version) {
         this.version = version;
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (!(o instanceof IssueTracker)) {
+            return false;
+        }
+
+        IssueTracker that = (IssueTracker) o;
+
+        if (!Objects.equals(name, that.name)) {
```

Comment:
Can be simplified to: 
return (Objects.equals(name, that.name) && Objects.equals(version, that.version));